website/content/api-docs/system/secrets-sync.mdx

```diff
@@ -826,3 +835,128 @@ $ curl \
     "auth": null
 }
 ```
+
+## Configure a custom GitHub app
+
+Use the `POST` operation with `sys/sync/github-apps` endpoint to configure a
+custom GitHub application for syncing secrets with GitHub repositories.
+
+
+| Method | Path                         |
+|:-------|:-----------------------------|
+| `POST` | `sys/sync/github-apps/:name` |
+
+
+### Parameters
+
+- `name` `(string: <required>)` - Specifies a custom name for the GitHub
+application that is used when configuring the destination.
+
+- `app_id` `(string: <required>)` - Specifies the GitHub application ID, provided by GitHub.
+
+- `private_key` `(string: <required>)` - Specifies the private key (either file contents or '@path/to/file')
+  used to authenticate with the GitHub application.
```

Comment:
@path/to/file only works due to a bit of CLI magic. The API does not accept a file reference.